tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -557,6 +557,13 @@ function install_artifacts() {
 			esac
 		fi
 
+		# TODO: this is a temporary change aiming to facilitate execution of CC-enabled GPU workloads in CI
+		# Next steps are to disable filesystem sharing and to use nydus guest pull for this handler.
+		# This will require adjustments to the pod manifest and deployment configuration.
+		if [[ "${shim}" == "qemu-nvidia-gpu-snp" ]]; then
+			sed -i -e 's/^shared_fs = "none"/shared_fs = "virtio-9p"/' "${kata_config_file}"
+		fi
+
```

Comment:
No, please, do not do this.

Changing this on kata-deploy.sh means that every single installation, inside or outside this CI, will come with virtio-9p enabled for SNP.

Do this change as part of the test, directly on the tests script, as part of the setup of each test.